app/views/shared/_application_information.html.erb

```diff
@@ -53,6 +53,14 @@
               </p>
             </td>
           </tr>
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell"><strong>Payment Amount:</strong></td>
+            <td class="govuk-table__cell">
+              <p class="govuk-body">
+                <%= number_to_currency (@planning_application.payment_amount.to_i/100), unit: '£' %>
```

Comment:
Not sure if you'd gain anything by moving the calculation to a helper, or if it's just adding code unnecessarily?